tests/integration/double_vote.go

```diff
@@ -0,0 +1,350 @@
+package integration
+
+import (
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	tmcrypto "github.com/cometbft/cometbft/crypto"
+	tmtypes "github.com/cometbft/cometbft/types"
+
+	testutil "github.com/cosmos/interchain-security/v3/testutil/crypto"
+	"github.com/cosmos/interchain-security/v3/x/ccv/provider/types"
+)
+
+// TestHandleConsumerDoubleVoting verifies that handling a double voting evidence
+// of a consumer chain results in the expected tombstoning and jailing the misbehaved validator
+func (s *CCVTestSuite) TestHandleConsumerDoubleVoting() {
+	s.SetupCCVChannel(s.path)
+	// required to have the consumer client revision height greater than 0
+	s.SendEmptyVSCPacket()
+
+	// create signing info for all validators
+	for _, v := range s.providerChain.Vals.Validators {
+		s.setDefaultValSigningInfo(*v)
+	}
+
+	consuValSet, err := tmtypes.ValidatorSetFromProto(s.consumerChain.LastHeader.ValidatorSet)
+	s.Require().NoError(err)
+	consuVal := consuValSet.Validators[0]
+	consuSigner := s.consumerChain.Signers[consuVal.Address.String()]
+
+	provValSet, err := tmtypes.ValidatorSetFromProto(s.providerChain.LastHeader.ValidatorSet)
+	s.Require().NoError(err)
+
+	provVal := provValSet.Validators[0]
+	provSigner := s.providerChain.Signers[provVal.Address.String()]
+
+	blockID1 := testutil.MakeBlockID([]byte("blockhash"), 1000, []byte("partshash"))
+	blockID2 := testutil.MakeBlockID([]byte("blockhash2"), 1000, []byte("partshash"))
+
+	// Note that votes are signed along with the chain ID
+	// see VoteSignBytes in https://github.com/cometbft/cometbft/blob/main/types/vote.go#L139
```

Comment:
Use a specific release reference here because line `139` at `vote.go` might show something else in the future.